### PR TITLE
PR: Bug and defects fix

### PR DESCRIPTION
**Pull request by Legatus727**

### CODE REVIEW

File: resources/JSON/Items.json [hunk 1]
```diff
@@ -41,7 +41,7 @@
   },
   "patientFile": {
     "name": "file",
-    "description": "You read the file... Patient is Jessica Smith. Held for mental health examination. Room 103. Door code is ####. The rest is illegible. Maybe her room will have an answer?",
+    "description": "You read the file... Patient is Jessica Smith. Held for mental health examination. Room 103. Door code is 9537. The rest is illegible. Maybe her room will have an answer?",
```
Comment (Legatus727) [inline resources/JSON/Items.json:44]: Assigned PIN

Comment (damon-m-deng) [reply resources/JSON/Items.json:44]: nice!

File: resources/JSON/roomsListNew.json [hunk 2]
```diff
@@ -85,30 +85,20 @@
...
     "directions": {
-      "east": "Key Pad",
+      "east": "Patient Room",
       "west": "Emergency Room"
     },
     "description": "This way to the patient room",
     "isLocked": false,
-    "items": []
-  },
-  "keyPad": {
```
Comment (Legatus727) [inline resources/JSON/roomsListNew.json:95]: Removed Key Pad as a room and implemented as a usable item in room (not able to pickup)

File: src/com/awakening/app/Game.java [hunk 3]
```diff
@@ -30,7 +27,8 @@ public class Game {
     public static Player player = new Player();
     public static NPC npc = new NPC();
     private static final Prompter prompter = new Prompter(new Scanner(System.in));
-    private List<String> approvedItems = new ArrayList<>(Arrays.asList("camera", "cellphone", "key", "journal", "batteries", "file", "bandages", "bandages", "paper-clip"));
+    private List<String> approvedItems = new ArrayList<>(Arrays.asList("camera", "cellphone", "key", "journal", "batteries", "file", "bandages", "bandages", "paper-clip", "press-pass"));
+    private List<String> usableItems = new ArrayList<>(List.of("key-pad"));
```
Comment (Legatus727) [inline src/com/awakening/app/Game.java:31]: usableItems list, can be expanded as we develop

Comment (damon-m-deng) [inline src/com/awakening/app/Game.java:31]: nice implementation

File: src/com/awakening/app/Game.java [hunk 4]
```diff
@@ -150,6 +148,13 @@ private void executeCommand(List<String> move) {
                     System.out.println(TextParser.RED + "Invalid command" + TextParser.RESET);
                 }
                 break;
+            case "use":
+                if (usableItems.contains(noun)){
+                    use(noun);
+                } else {
+                    System.out.println(TextParser.RED + "Cannot use that" + TextParser.RESET);
+                }
+                break;
```
Comment (Legatus727) [inline src/com/awakening/app/Game.java:157]: use case for checking player command 'use'

File: src/com/awakening/app/Game.java [hunk 5]
```diff
@@ -220,6 +227,27 @@ private void pickUp(String noun) {
...
+
+        if (keyEntry.equalsIgnoreCase("9537")) {
+            RoomMap.RoomLayout currentRoom = player.getCurrentRoom();
+            RoomMap.RoomLayout nextRoom = world.getRoom(currentRoom.getDirections().get("east"));
+
+            nextRoom.setLocked(false);
+            System.out.println("The key-pad chimes and turns green.");
+        }
+    }
+
```
Comment (damon-m-deng) [inline src/com/awakening/app/Game.java:250]: love it

Comment (Legatus727) [inline src/com/awakening/app/Game.java:250]: Basic use method, needs refactoring for anything beside 'key-pad'

File: src/com/awakening/app/Game.java [hunk 6]
```diff
@@ -259,8 +287,5 @@ private void generateItems() {
         }
 
     }
-
-    private void startGame() {
-    }
```
Comment (Legatus727) [inline src/com/awakening/app/Game.java:264]: unused

**Review by damon-m-deng — APPROVED** [review]
very nice work!

File: resources/JSON/Items.json (hunk 1 above)
Comment (javidur) [reply resources/JSON/Items.json:44]: I like it

**Review by javidur — APPROVED** [review]

**Pull request merged**